spotify_dl/spotify_dl.py

```diff
@@ -40,14 +40,26 @@ def spotify_dl():
                         default=['bestaudio/best'])
     parser.add_argument('-m', '--skip_mp3', action='store_true',
                         help='Don\'t convert downloaded songs to mp3')
-    parser.add_argument("--url", action="store_true", help="Convert playlist link to uri first")
+    parser.add_argument('-l', '--url', action="store",
+                        help="Spotify Playlist link URL")
```

Comment:
`store` doesn't seem right for this variable name. I am not sure.

cc @SathyaBhat